devops/docker/debian11-arm/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM multiarch/qemu-user-static:x86_64-arm as qemu
+FROM arm32v7/debian:bullseye
```

Comment:
Use secure mirror `mcr.microsoft.com/mirror/docker/library/debian:bullseye`

Reply:
For this and others: no such mirror exists for arm32/46 images in MCR.

Reply:
I see. We should onboard these if they are not present, see https://eng.ms/docs/cloud-ai-platform/azure-core/azure-management-and-platforms/containers-bburns/microsoft-container-registry/microsoft-artifact-registry-also-known-as-microsoft-container-registry-docs/onboardingdocs/newimageonboarding